### DISCUSSION TITLE
X10 gift boxes for those that want them...

### DISCUSSION THREAD

**Discussion by Bromaynard** [HeyuX10Automation/heyu#66] [General]
I am moving and decided to remove all my X10 stuff from my home, and do not have a need for it anymore.  If anyone wants a couple boxes of stuff...

Box 1:
RCS TX15 Thermostat - 1
XTB-523 X10 PLC Interface (TW523/PSC05 replacement) - 1
Rain8 X10 8-Zone Sprinkler controller w/PSC05 - 1
Icon Remote - 1
IR543 Infrared Command Center - 1
ELK ESM1 X10 Signal Meter - 1

CM11A - 2
Firecracker CM17A - 3
PSC05 2-way powerline interface - 2
Transceiver modules - 6
Appliance modules (3 prong) - 13
Lamp modules (2 prong) - 5
UM506 Universal dry contact relay module - 1
SC546A Remote Chime - 1
HR12A 16 device remotes - 2
CR12A 16 channel (Camera) remote -1
SS13A Stick-a-switch (Stick on wall switch) - 2
4 channel keychain remotes - 2
KR19a 2 channel keychain remote - 1
MS14A Motion Detectors -4
MS16A ActiveEye (Adjustable) Motion Detectors - 2

Box 2:
PAO11 SuperSocket Receptacles - 5
Leviton 6381-Wi X10 Decorator Dimmer Switch - 3
Leviton 6294-W X10 Decorator SLAVE Switch - 1
RWS17 X10 Decorator Dimmer Switch - 3
WS12A Decorator Dimmer Switch - 5
WS13A X10 Decorator Switch - 1
WS14A X10 Decorator Dimmer SLAVE Switch - 2

I also have a phase coupler I still need to remove from my breaker panel, but I need to get an electrician out as I am a bit older now and hands and eyes do not work like they used to, but I will include those as well.  I do not want to just toss them, so all are free, and may even pay for the shipping (my spouse works in a mail room) depending on how much it costs.  Just let me let me know.  ;)

**Comment by cagivacode** [comment]
I would be interested....my house has been x10 for 30 years...but some of the devices are starting to malfunction. I am also adding on to the existing system because I went to a time-of-day pricing for electric and have to shift as much usage from "peak" to "super off peak" to make it cost effective.

I would be happy to pay for shipping....

roger

**Reply by Bromaynard** [reply]
@cagivacode ... Sorry for the delayed response.  Apparently, my browser on my phone is old enough that it would not allow me to reply.  :(

I did not expect a response this fast as well, but 1st come, 1st served.  ;)

Is there a way to private message on github?  If not, I can create a temporary email to post so we can communicate that way.

I still need to remove the Phase Coupler from my breaker panel, and I found another receptacle in use in my garage I forgot about.  However, we can make arrangements within the next couple week to figure out the logistics, and hopefully we can get this stuff to you before Christmas?!  :D

**Comment by cagivacode** [comment]
They got rid of the private message thing....use strap at xxx dot com.
I have my own server and set that up as a spam trap anyway.
No rush...will be interesting to see if the Phase Coupler works better than the radio shack cap I installed about 35 years ago :)

**Reply by Bromaynard** [reply]
OK.  Email sent.  It might appear to come from a localhost server, but the domain is valid and spf should pass.  I do not have DMARC setup, because I only allow email from accounts on the local host so hopefully your spam service is not too strict and bounces it.  Look forward to hearing back.

**Comment by davepetlive** [comment]
I hate that I missed the early Christmas.
X10 forever.

On Sunday, November 27, 2022, Bromaynard ***@***.***> wrote:
> OK. Email sent. It might appear to come from a localhost server, but the
domain is valid and spf should pass. I do not have DMARC setup, because I
only allow email from accounts on the local host so hopefully your spam
service is not too strict and bounces it. Look forward to hearing back.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>

**Reply by Bromaynard** [reply]
Just hang out... There are plenty of us old farts moving into homes, that cant take the X10 with us, and it is too difficult for most of the next generations to troubleshoot.  I acquired probably 1/5 of my devices from another old timer.  X10 is still great, especially with heyu2.  I have been using it for 25+ years.  Rock solid after a few months of hammering out the noise issues?!  👍
